jetty-jaspi/pom.xml

```diff
@@ -63,38 +63,28 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty.orbit</groupId>
-      <artifactId>javax.security.auth.message</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>javax.xml.bind</groupId>
-      <artifactId>jaxb-api</artifactId>
-      <version>2.3.1</version>
-      <scope>test</scope>
+      <groupId>jakarta.authentication</groupId>
+      <artifactId>jakarta.authentication-api</artifactId>
+      <version>2.0.0</version>
```

Comment:
This needs to be defined in the top level `pom.xml`, and the existing declaration for `org.eclipse.jetty.orbit|javax.security.auth.message `removed.
Also the `jetty-home/pom.xml `also mentions the old `javax.security.auth.message` and `javax.activation` artifacts and should also be removed and replaced with the new jakarta jars instead.
The `jetty-maven-plugin/src/it/jetty-start-war-mojo-it/pom.xml` also is relying on `javax.activation` and needs to be upgraded - this probably depends on upgrading to a newer version of the glassfish `bean-validation-webapp`.

Reply:
* removing org.eclipse.jetty.orbit|javax.security.auth.message
* removing RC3 activation-api version referenced in jetty-osgi/test-jetty-osgi

* The glassfish bean-validation-webapp seems to have been ditched 3 years ago so that's going to be hard.
Seems to have happened the jersey repository moved from https://github.com/javaee/jersey/tree/master/examples to https://github.com/eclipse-ee4j/jersey, 
d (https://github.com/eclipse-ee4j/jersey/commit/065d21270092d6bed7b2dabbe1f65e0933372b80)
Would suggest this would be better left to a separate issue

Reply:
@olamy should we update the bean-validation-webapp that the jetty-maven-plugin tests are using???

Reply:
I presume the issue of the bean-validation-webapp should be opened as a separate issue, marking conversation resolved